raphtory/src/db/view_api/internal.rs

```diff
@@ -650,66 +651,85 @@ pub trait GraphViewInternalOps {
     ) -> Box<dyn Iterator<Item = LocalVertexRef> + Send>;
 }
 
+pub trait WrappedGraph {
```

Comment:
What about `AsRef<G>`

Reply:
Does not work because of unconstrained parameter when used with generic graph